nucleus/payara-modules/hazelcast-bootstrap/src/main/java/fish/payara/nucleus/hazelcast/HazelcastCore.java

```diff
@@ -127,13 +127,13 @@
 
     @Inject
     ServerContext context;
-    
+
```

Comment:
In future watch out for these. IDE's like to constantly add or remove these whitespace characters and it's mildly annoying having to filter them out, particularly when it's done to a file with no other changes!

Reply:
I have my IDE set to remove by default. They should be removed. You can always go git diff w/o whitespace in the settings or `git diff -b` that's what I do :)